### PR TITLE
446 - rule description in graph

### PR DESCRIPTION
**Pull request by klinki**

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

File: reporting/impl/src/main/java/org/jboss/windup/reporting/ruleexecution/GetRuleExecutionResults.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public Object exec(@SuppressWarnings("rawtypes") List arguments) throws Template
         StringModel stringModelArg = (StringModel) arguments.get(0);
         AbstractRuleProvider ruleProvider = (AbstractRuleProvider) stringModelArg.getWrappedObject();
         List<RuleExecutionInformation> result = RuleExecutionResultsListener.instance(this.event).getRuleExecutionInformation(ruleProvider);
-        ExecutionStatistics.get().begin(NAME);
+        ExecutionStatistics.get().end(NAME);
```
Comment (jsight) [inline reporting/impl/src/main/java/org/jboss/windup/reporting/ruleexecution/GetRuleExecutionResults.java]: Good catch. :)

**Comment by mareknovotny** [comment]
it seems to me ok, my local runs and passing

**Comment by mareknovotny** [comment]
retest

**Comment by mareknovotny** [comment]
so tests are not passing due our feedback form Caused by: net.sourceforge.htmlunit.corejs.javascript.EcmaError: ReferenceError: "$" is not defined. (file:/var/lib/jenkins/workspace/windup-pr-builder/tests/target/Windup/windupgraph_cX99py/reports/resources/js/navbar.js#1)

**Comment by klinki** [comment]
retest

**Comment by mareknovotny** [comment]
retest

testing setting of CI build status

**Comment by mareknovotny** [comment]
retest

**Pull request merged**